### ISSUE TITLE
Loss of Ad-Blocking Capability when Connected using IPv6

### ISSUE THREAD

**Issue by christantoan** [AdguardTeam/AdguardForAndroid#4224]
### Issue Details

- **AdGuard version:**
  - v3.6.9 (48)
- **Filtering mode:**
  - HTTP Proxy (Auto)
- **Device:**
  - OnePlus 7 Pro
- **Operating system and version:**
  - Oxygen OS 11.0.9.1.GM21BA
- **Root access:**
  - Yes

### Expected Behavior

Chrome and other webview pages still have their ads blocked on all websites when using certain geographic VPN servers and AdGuard's Local HTTP Proxy mode.

### Actual Behavior

Chrome and other webview pages still have their ads loaded on some websites when using certain geographic VPN servers and AdGuard's Local HTTP Proxy mode.

### Screenshots

<details><summary>Screenshot: Google News Ad Loaded (https://github.com/AdguardTeam/AdguardFilters/issues/127893)</summary>

![image](https://user-images.githubusercontent.com/7300310/186139487-7349c78c-86f8-479f-9200-a09b5bc7e7ea.png)

</details>

<details><summary>Screenshot: Chrome Ad Loaded in Google Search result page (https://github.com/AdguardTeam/AdguardFilters/issues/127895)</summary>

![image](https://user-images.githubusercontent.com/7300310/186139746-00bf63f7-4ae1-4baa-a8d5-cf163605f345.png)

</details>

<details><summary>Screenshot: Filtering Log still shows that Chrome is still filtered but not all pages are detected</summary>

![image](https://user-images.githubusercontent.com/7300310/186140135-dba1e16a-64dd-46c1-9e71-3a6ee988271d.png)

</details>

### Additional Information

Using Local HTTP Proxy in combination with my VPN provider Mullvad (only on some geographic servers; I tested SG, NL, and JP servers; only SG servers have the problem). Using AdGuard's Local VPN mode and SG servers also solved the problem (I use WireGuard kernel mode so I can use two VPN providers at the same time).

It used to work fine until a few weeks ago.

Additional settings details can be found in the issues above.
Additional system details can be found in ticket number 631512.

**Comment by christantoan** [comment]
Update: Disabling IPv6 seems to also fix the problem irrespective of the filtering mode used.

**Comment by christantoan** [comment]
Tried updating to 3.6.51 and latest nightly but still not resolved.

**Comment by maxikuzmin** [comment]
@christantoan we're looking into the problem. Please wait

**Comment by maxikuzmin** [comment]
@christantoan duplicate of <https://github.com/AdguardTeam/AdguardForAndroid/issues/3901>
Please just wait version 4.0
